api/v1alpha4/nnfsystemstorage_types.go

```diff
@@ -83,6 +83,11 @@ type NnfSystemStorageSpec struct {
 	// +kubebuilder:default:=raw
 	Type string `json:"type,omitempty"`
 
+	// Shared will create one allocation per Rabbit rather than one allocation
+	// per compute node.
+	// +kubebuilder:default:=true
+	Shared bool `json:"shared"`
+
 	// StorageProfile is an object reference to the storage profile to use
```

Comment:
Where does the user set this? Gitops?

Reply:
Anywhere you can create a NnfSystemStorage resource. kubectl, gitops, etc.